packages/@o3r/artifactory-tools/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@o3r/artifactory-tools",
+  "version": "0.0.0-placeholder",
+  "description": "Various artifactory tools",
+  "keywords": [
+    "otter",
+    "otter-module"
+  ],
+  "scripts": {
+    "nx": "nx",
+    "ng": "yarn nx",
+    "build": "yarn nx build artifactory-tools",
+    "postbuild": "patch-package-json-main",
+    "prepare:publish": "prepare-publish ./dist",
+    "prepare:build:builders": "yarn cpy 'schematics/**/*.json' dist/schematics && yarn cpy 'collection.json' dist",
+    "build:builders": "tsc -b tsconfig.builders.json --pretty && yarn generate-cjs-manifest"
+  },
+  "bin": {
+    "o3r-artifact-cleaner": "./dist/src/cli/artifact-cleaner.js",
```

Comment:
`dist` directory does not exist in the published package. Is it removed from the path by the `patch-package-json-main` script

Reply:
`dist` is removed in the dist folder but needed in local